src/main/java/org/terasology/signalling/componentSystem/SignalNetworkNode.java

```diff
@@ -26,17 +26,24 @@
         PRODUCER, CONSUMER, CONDUCTOR
     }
 
+    // The node type
```

Comment:
This does not add anything. you probably don't need anything in terms of javadocs for this line.

Reply:
Definitely agree with Michael. The variable "type" is explanatory enough. 

Always remember, the aim is not to add comments to every line. There are two contradictory factors at play - adding more comments to better explain the code, and adding fewer comments in favour of self-explanatory code. To balance the two, only add comments where the code is (even slightly) hard to understand - after you've tried and failed to simplify the code.